File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/OrsonAiEmotionsResponse.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class OrsonAiEmotionsResponse 
{
    public string RequestId { get; set; }
    public int ResponseCode { get; set; }
    public string ResponseRaw { get; set; }
    public string Status { get; set; }
    public string Error { get; set; }
    public List<string> AudioEmotions { get; set; }
    public List<OrsonAiVisualEmotionResponse> VisualEmotions { get; set; }
}

File: server-scala-play-server/app/model/OrsonAiEmotionsResponse.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for OrsonAiEmotionsResponse.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class OrsonAiEmotionsResponse(
  requestId: Option[String],
  responseCode: Option[Int],
  responseRaw: Option[String],
  status: Option[String],
  error: Option[String],
  audioEmotions: Option[List[String]],
  visualEmotions: Option[List[OrsonAiVisualEmotionResponse]]
)

object OrsonAiEmotionsResponse {
  implicit lazy val orsonAiEmotionsResponseJsonFormat: Format[OrsonAiEmotionsResponse] = Json.format[OrsonAiEmotionsResponse]
}

